### PR TITLE
research paper proposal added

### PR DESCRIPTION
**Pull request by yestmd**
Thanks Professor! @enkeboll

### CODE REVIEW

File: paper/proposal.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Research Paper Proposal
+
+## Title: A potential key to open the chamber of secrets of neurocognitive functions by applying Thunder---a big data tools in neuroscience field
```
Comment (enkeboll) [inline paper/proposal.md:3]: wait-- I thought the Chamber of Secrets was a bad thing!?

**Review by enkeboll — APPROVED** [review]
this is an amazing topic.  I'm excited about it already.  I think to the best that you can, I'd love to see you write a how-to about Thunder in Databricks, or something similar.  You might not have time to write about "current usage of Thunder tools", but that would be just fine.

File: paper/proposal.md (hunk 1 above)
Comment (yestmd) [reply paper/proposal.md:3]: Thanks Professor! 
I can change the Chamber of Secrets in the title. Maybe write it as "open the door of", if that is OK. 
Thanks again.

Comment (enkeboll) [reply paper/proposal.md:3]: I was only making a Harry Potter joke! Call it whatever you like

**Pull request merged**